src/bundleList.js

```diff
@@ -127,6 +131,13 @@ function initBundleBuckets(roles) {
 function sortBundleByBuckets(roles, bundle, bundleBuckets) {
   roles.forEach((role) => {
     if (bundle.indexOf('-' + role + '-') !== -1) {
+
+      // skip invalid bundle names
+      if( !validBundleRegex.test( bundle ) ){
+        console.error( 'skip bundle', bundle );
```

Comment:
@dianashk right. My thinking is this message should go to `stdout` as it's nice to be able to scan the output of `stderr` for only messages that would indicate a crash or other critical error.

@missinglink. How about something like "skipping older bundle: ${bundle_name}". I'm imagining (and tell me if this doesn't make sense) a user seeing that the downloader is skipping lots of data without giving a reason, and they then assume that the downloader is choosing not to give them data that they think they want.